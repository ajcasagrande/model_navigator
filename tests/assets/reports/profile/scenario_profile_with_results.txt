ProfileEvent.PROFILING_STARTED {}
ProfileEvent.RUNTIME_PROFILING_STARTED {'name': 'python on eager'}
ProfileEvent.RUNTIME_PROFILING_RESULT {'result': ProfilingResult(batch_size=1, avg_latency=1, std_latency=1,p50_latency=1,p90_latency=1, p95_latency=1,p99_latency=1,throughput=1000,request_count=1)}
ProfileEvent.RUNTIME_PROFILING_RESULT {'result': ProfilingResult(batch_size=2, avg_latency=2, std_latency=2,p50_latency=2,p90_latency=2, p95_latency=2,p99_latency=2,throughput=1000,request_count=1)}
ProfileEvent.RUNTIME_PROFILING_FINISHED {}
ProfileEvent.RUNTIME_PROFILING_STARTED {'name': 'navigator on optimized'}
ProfileEvent.RUNTIME_PROFILING_RESULT {'result': ProfilingResult(batch_size=1, avg_latency=1, std_latency=1,p50_latency=1,p90_latency=1, p95_latency=1,p99_latency=1,throughput=1000,request_count=1)}
ProfileEvent.RUNTIME_PROFILING_RESULT {'result': ProfilingResult(batch_size=2, avg_latency=1, std_latency=1,p50_latency=1,p90_latency=1, p95_latency=1,p99_latency=1,throughput=2000,request_count=1)}
ProfileEvent.RUNTIME_PROFILING_RESULT {'result': ProfilingResult(batch_size=4, avg_latency=1, std_latency=1,p50_latency=1,p90_latency=1, p95_latency=1,p99_latency=1,throughput=4000,request_count=1)}
ProfileEvent.RUNTIME_PROFILING_FINISHED {}
ProfileEvent.RUNTIME_PROFILING_STARTED {'name': 'onnx on OnnxCUDA'}
ProfileEvent.RUNTIME_PROFILING_ERROR {}
ProfileEvent.PROFILING_FINISHED {}
