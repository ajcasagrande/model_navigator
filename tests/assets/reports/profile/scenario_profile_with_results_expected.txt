Profiling started
python on eager: profiling ...
python on eager: Batch      1, Throughput:    1000.00 [infer/sec], Avg Latency:       1.00 [ms]
python on eager: Batch      2, Throughput:    1000.00 [infer/sec], Avg Latency:       2.00 [ms]
python on eager: profiling OK
navigator on optimized: profiling ...
navigator on optimized: Batch      1, Throughput:    1000.00 [infer/sec], Avg Latency:       1.00 [ms]
navigator on optimized: Batch      2, Throughput:    2000.00 [infer/sec], Avg Latency:       1.00 [ms]
navigator on optimized: Batch      4, Throughput:    4000.00 [infer/sec], Avg Latency:       1.00 [ms]
navigator on optimized: profiling OK
onnx on OnnxCUDA: profiling ...
onnx on OnnxCUDA: profiling FAILED
Profiling finished
                            Profiling status                             
┏━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┓
┃ Model on Runtime       ┃ Status ┃ Batch  ┃ Throughput  ┃ Avg Latency  ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━━┩
│ python on eager        │   OK   │      1 │    1000.00  │       1.00   │
│                        │        │      2 │    1000.00  │       2.00   │
├────────────────────────┼────────┼────────┼─────────────┼──────────────┤
│ navigator on optimized │   OK   │      1 │    1000.00  │       1.00   │
│                        │        │      2 │    2000.00  │       1.00   │
│                        │        │      4 │    4000.00  │       1.00   │
├────────────────────────┼────────┼────────┼─────────────┼──────────────┤
│ onnx on OnnxCUDA       │ FAILED │        │             │              │
└────────────────────────┴────────┴────────┴─────────────┴──────────────┘
